src/datasource/zabbix/zabbix.ts

```diff
@@ -208,32 +208,21 @@ export class Zabbix implements ZabbixConnector {
    * ```
    */
   testDataSource() {
```

Comment:
I don't think we need this anymore. I think when we hit `save&test` button in the config editor it just calls `CheckHealth` endpoint.

Reply:
This one we need to be able to fetch the dbConnection status when a datasource is selected for `dbConnectionEnabled`